actionpack/lib/action_dispatch/middleware/static.rb

```diff
@@ -13,15 +13,13 @@ module ActionDispatch
   # located at `public/assets/application.js` if the file exists. If the file
   # does not exist, a 404 "File not Found" response will be returned.
   class FileHandler
-    def initialize(root, cache_control, index = 'index')
+    def initialize(root, index = 'index', headers: )
```

Comment:
oh, I didn't know that `ActionDispatch::FileHandler` is a public class. I don't think it's ok to break it.